c2corg_api/emails/email_service.py

```diff
@@ -1,3 +1,5 @@
+import transaction  # NOQA
```

Comment:
Yes with demov6 SMTP server, and I've received the email, I've change my password !!!